<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EthernetConnectionWidgetForm</class>
 <widget class="QWidget" name="EthernetConnectionWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="address_lineedit_">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>192.168.0.10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="port_spinbox_">
     <property name="maximum">
      <number>65536</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="connect_button_">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Connect to device</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="EthernetConnectionWidget.qrc">
       <normaloff>:/EthernetConnectionWidget/connect_icon</normaloff>:/EthernetConnectionWidget/connect_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="disconnect_button_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Disconnect from device</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="EthernetConnectionWidget.qrc">
       <normaloff>:/EthernetConnectionWidget/disconnect_icon</normaloff>:/EthernetConnectionWidget/disconnect_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EthernetConnectionWidget.qrc"/>
 </resources>
 <connections/>
</ui>
